src/index.html

```diff
@@ -1,14 +1,163 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Human rights</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <h1 class="h1">Universal Declaration of Human Rights</h1>
+    <h2 class="h2">Preamble</h2>
+    <p>Whereas recognition of the inherent dignity and of the equal and inalienable
```

Comment:
```suggestion
    <p>
      Whereas recognition of the inherent dignity and of the equal and inalienable
```
read the checklist, you need to add one more level of indentations after you opened the tag